src/plugins/docusaurus-static-fonts/index.js

```diff
@@ -0,0 +1,25 @@
+const webpack = require("webpack")
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
+
+// eslint-disable-next-line
+module.exports = function (context, options) {
+  return {
+    name: "docusaurus-static-fonts",
+    // eslint-disable-next-line
+    configureWebpack(config, isServer, utils) {
```

Comment:
Shouldn't these files just be statically served, as a static asset? Does webpack need to process them?

Reply:
No it doesn't, but I can't figure out how to tell docusaurus to NOT modify them. As soon as CSS uses `url` it bundles the files and ads content hashes. I will probably just reference the file with the content hash in preload as the contents of the file won't likely change anyways.

Reply:
Ahh, webpack processes them. Okay I understand.. maybe we can just host these fonts on a static file server or using cloudflare on `ory.sh`

Reply:
We could, but the CSS bundler does its thing and replaces the paths